chatlas/_content.py

```diff
@@ -224,9 +232,9 @@ def __repr__(self, indent: int = 0):
             res += f" error='{self.error}'"
         return res + ">"
 
+    # The actual value to send to the model
     def get_final_value(self) -> str:
-        value, _language = self._get_value_and_language()
-        return value
+        return self._get_value()
```

Comment:
This is an intentional, and somewhat orthogonal, change in behavior that was introduced in #30 where we started sending a `pformat()` version if the value can be loaded as JSON. Now, instead, we always send the `str()`'d result for consistency sake (as we did before #30). I _think_ @schloerke's intention in #30 was to just get better formatting of tool calls when `echo="all"`, but I might be wrong (sorry for not catching this earlier).

Reply:
Moving to #70